### DISCUSSION TITLE
ImportError: cannot import name 'COMPUTE_KIND_TAG'

### DISCUSSION THREAD

**Discussion by dlambert-personal** [dagster-io/dagster#22755] [Q&A] [answered]
As soon as I reference dagster_snowflake_pandas - ex:

`from dagster_snowflake_pandas import SnowflakePandasIOManager`

I see the following error in my local dev environment:

ImportError: cannot import name 'COMPUTE_KIND_TAG' from 'dagster._core.storage.tags' (/home/dlambert/.local/lib/python3.10/site-packages/dagster/_core/storage/tags.py)

It looks like the line referenced in /dagster_snowflake/ops.py may have changed ~3 weeks ago.

I tried pip uninstall dagster and pip install dagster to make sure I wasn't missing an update.  Are there additional modules or updates I need to reference to pull in recent changes?

**Comment by garethbrickman** [comment, marked as answer]
Installing your project dependencies, which should include `dagster` and `dagster_snowflake_pandas`, should be all you need to do to get the latest updated versions. Could you double-check in your `setup.py` or `requirements.txt` that you're not setting an older version? 

You can also verify the versions installed by running: `pip freeze | grep dagster`

**Reply by dlambert-personal** [reply]
That looks like the problem.   Versions indicate dagster 1.7.6:

dagster==1.7.6
dagster-cloud==1.7.10
dagster-cloud-cli==1.7.10
dagster-duckdb==0.23.10
dagster-duckdb-pandas==0.23.10
dagster-graphql==1.7.6
dagster-pipes==1.7.6
dagster-snowflake==0.23.10
dagster-snowflake-pandas==0.23.10
dagster-webserver==1.7.6

I'm not positive why I'm stuck on 1.7.6 - it's not indicated in setup.py with a specific version, but I'll poke at this a bit and see if I can get the upgrade to stick.  Thanks for the pointer.

**Reply by dlambert-personal** [reply]
> Installing your project dependencies, which should include `dagster` and `dagster_snowflake_pandas`, should be all you need to do to get the latest updated versions. Could you double-check in your `setup.py` or `requirements.txt` that you're not setting an older version?
> 
> You can also verify the versions installed by running: `pip freeze | grep dagster`

This was what I needed to troubleshoot - I had a local install that was stuck at 1.7.6 - I needed to get that cleaned up to pick up the new version.  Thanks for your help.
